README.md

```diff
@@ -6,8 +6,8 @@ JetPack code and documentation is distributed under the Apache 2 license. Contr
 
 JetPack is an innovative automation package that is used to configure the infrastructure hardware and OpenStack software in a fully automated fashion. The toolkit includes scripts from Dell EMC that work with Red Hat OpenStack Platform Director to automate the deployment process, not only to save time but to ensure the process is reliable and repeatable each time.
 
-Please reference the current Deployment Guide for Dell EMC NFV Ready Architecture for Red Hat Software v13.0 for installation and configuration at:
-https://github.com/dsp-jetpack/JetPack/blob/master/doc/deployment_guide.md
+Please reference the current Deployment Guide for Dell EMC NFV Ready Architecture for Red Hat Software v13.2 for installation and configuration at:
```

Comment:
Should be "Dell EMC Ready Architecture For Red Hat OpenStack Platform Software Deployment Guide v13.2